packages/atlas-service/package.json

```diff
@@ -82,10 +82,10 @@
     "electron": "^25.9.7",
     "hadron-ipc": "^3.2.5",
     "keytar": "^7.9.0",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "^2.7.0",
```

Comment:
Accidentally ran depalign with autofix after merging with main, doesn't hurt to update those I guess?

Reply:
I wonder if we should include this in the other jobs that run automatically 🤔

Reply:
Yeah, good idea, maybe with the authors / third party file stuff, let me do just that